object/message.go

```diff
@@ -334,3 +346,14 @@ func GetPaginationMessage(owner string, offset, limit int, field, value, sortFie
 
 	return messages, nil
 }
+
+func GetMessageTextTokenCount(modelName string, text string) (int, error) {
+	tokenCount, err := model.GetTokenSize(modelName, text)
+	if err != nil {
+		tokenCount, err = model.GetTokenSize("gpt-3.5-turbo", text)
```

Comment:
When enter this if?

Reply:
modelname is empty or not supported by tiktoken